osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -234,6 +244,16 @@ private void initializeChannels()
                     addChannel(channels.Find(c => c.Name == @"#lazer"));
                     addChannel(channels.Find(c => c.Name == @"#osu"));
                     addChannel(channels.Find(c => c.Name == @"#lobby"));
+
+                    channelSelection.OnRequestJoin = addChannel;
+                    channelSelection.Sections = new[]
+                    {
+                        new ChannelSection
+                        {
+                            Header = @"ALL CHANNELS",

```

Comment:
no @ prefix. also for cases of upper-case, it's best to use `.ToUpper` and leave the actual string in title case I think.